### ISSUE TITLE
UI fixes and debugs

### ISSUE THREAD

**Issue by jairabp** [kalo-stems/kalo-kode#38]
Do all work in a branch called issue-38

BEFORE STARTING, MAKE SURE ALL BRANCHES ARE MERGED INTO MAIN BEFORE CREATING BRANCH ISSUE-38 TO ENSURE EVERYTHING IS DEBUGGED AND UI-FIXED

- add placeholders in all add and edit form pages to Students, Company Profile, and Job pages
- adjust lengths of textboxes and lists to make it easier on the eye 
- clean up simple issues such as black text on dark bg
- ensure all links work
